modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -15,6 +15,29 @@ New features are added to the language continuously, and occasionally, some feat
 
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
+
+[[cypher-deprecations-additions-removals-5.19]]
+== Neo4j 5.19
+
+=== New features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:syntax[]
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+FINISH
+----
+| Added the xref:clauses/finish.adoc[FINISH], which defines a query to have no result set.
```

Comment:
"defines a query to have no result", while in line 48 of `modules/ROOT/pages/clauses/index.adoc` it says "defines that a query has no result set"

which one suits FINISH better? use in both cases :)

Reply:
Both are meant to say the same. There is no "fits better". Question is which is better English? I guess the latter. Agreed?

Reply:
let's go with "defines a query to have no result"
i asked @fiquick for native speaker input (thanks! :) )